lib/cli/test/fixtures/sfc_vue/.postcssrc.js

```diff
@@ -1,8 +0,0 @@
-// https://github.com/michael-ciniawsky/postcss-load-config
```

Comment:
For some reason, those 2 files kept producing noise even after adding an ignore pattern